src/main/js/PipelineEditorLink.jsx

```diff
@@ -15,7 +15,9 @@ class PipelineEditorLink extends React.Component {
         const href = Paths.rest.apiRoot() + '/organizations/' + pipeline.organization + '/pipelines/' + folder + '/';
         pipelineService.fetchPipeline(href, { useCache: true })
         .then(pipeline => {
-            if (pipeline._class === 'io.jenkins.blueocean.blueocean_github_pipeline.GithubOrganizationFolder') {
+            if (pipeline._class === 'io.jenkins.blueocean.blueocean_github_pipeline.GithubOrganizationFolder' ||
```

Comment:
Yes, this is how this should have been done. But it's ok as the list is not going to grow. In fact it is going to end up only being mult branch pipeline as we are going to remove org folder. Anyways  we can do capability way in other PR or if you tell me how capabilities are checked in JS code of editor I could do that.